api/client/examples/list-resources/go.mod

```diff
@@ -1,22 +1,25 @@
 module github.com/deckhouse/virtualization/api/client/examples/list-resources
 
-go 1.21.4
+go 1.22.7
 
 require (
 	github.com/deckhouse/virtualization/api v0.0.0-20240322104947-2d492906a8b2
 	github.com/spf13/pflag v1.0.5
 	k8s.io/apimachinery v0.29.3
 )
 
+require (
+	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang/protobuf v1.5.4 // indirect
+)
+
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.11.0 // indirect
 	github.com/go-logr/logr v1.4.1 // indirect
 	github.com/go-openapi/jsonpointer v0.19.6 // indirect
 	github.com/go-openapi/jsonreference v0.20.2 // indirect
 	github.com/go-openapi/swag v0.22.3 // indirect
-	github.com/gogo/protobuf v1.3.2 // indirect
-	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/google/gnostic-models v0.6.8 // indirect
```

Comment:
Why separate require? Is it after `go mod tidy` or you change it accidentally?

Reply:
Is it after go mod tidy, I'll try move them back